src/util.js

```diff
@@ -295,12 +294,17 @@ export const isFeatureState = (
 }
 
 
-export const urlsFromRepoCtxFeature = (repoCtxFeature) => {
-  if (!repoCtxFeature || !repoCtxFeature.isAvailable) {
+export const urlsFromOcmRepositoryCfgsFeature = (ocmRepositoryCfgsFeature) => {
+  if (!ocmRepositoryCfgsFeature || !ocmRepositoryCfgsFeature.isAvailable) {
     return []
   }
 
-  return [OCM_REPO_AUTO_OPTION, ...repoCtxFeature.cfg.repoContexts.map(rc => rc.baseUrl)]
+  return [...ocmRepositoryCfgsFeature.ocm_repository_cfgs.reduce((repositories, ocmRepositoryCfg) => {
+    if (ocmRepositoryCfg.repository) repositories.add(ocmRepositoryCfg.repository)
```

Comment:
if this is about backwards compatibility, please add a brief comment stating so :)